google/cloud/bigtable/internal/default_row_reader_test.cc

```diff
@@ -811,6 +823,40 @@ TEST_F(DefaultRowReaderTest, ReverseScanResumption) {
   EXPECT_THAT(StatusOrRowKeys(reader), ElementsAre(IsOkAndHolds("r3")));
 }
 
+TEST_F(DefaultRowReaderTest, BigtableCookies) {
+  auto mock = std::make_shared<MockBigtableStub>();
+  EXPECT_CALL(*mock, ReadRows)
+      .WillOnce([this](auto context, auto const&,
```

Comment:
Do you have to pass this or can you pass only what you use?

Reply:
I use the member variable `metadata_fixture_` inside the lambda. We have to capture `this` to use any of the member variables. Think of them as like `this->metadata_fixture_`

Reply:
Ah I see. Seems like a weird way to do it, but 🤷